include/glow/Runtime/Provisioner/Provisioner.h

```diff
@@ -51,6 +54,13 @@ class Provisioner final {
   /// Padding factor to account for generated code size. Should be greater
   /// than 1.0.
   const float NETWORK_PADDING_FACTOR = 1.1;
+
+  /// Vector of DeviceManagers.
```

Comment:
Is this? Maybe something like "the list of available devices, created at setup .... "